ui/src/app/pages/admin/admin-users.tsx

```diff
@@ -133,50 +137,173 @@ export const AdminUsers = withUserProfile()(class extends React.Component<
     return new Date(date).toString().split(' ').slice(1, 5).join(' ');
   }
 
-  convertProfilesToFields(profiles: Profile[]) {
-    return profiles.map(p => ({
-      ...p,
-      name: <a onClick={() => navigate(['admin', 'users', usernameWithoutDomain(p.username)])}>
-        {p.familyName + ', ' + p.givenName}
+  getAccessModuleString(user: AdminTableUser, key: string) {
+    const completionTime = user[key + 'CompletionTime'];
+    const bypassTime = user[key + 'BypassTime'];
+    if (completionTime) {
+      return '✔';
+    } else if (bypassTime) {
+      return 'B';
```

Comment:
Some of [these icons](https://www.google.com/search?q=bypass+icon&rlz=1C5CHFA_enUS831US832&sxsrf=ALeKk00xneNgPRhA30xCN_E2QJI0sanKCQ:1601645825260&source=lnms&tbm=isch&sa=X&ved=2ahUKEwjEsuHTg5bsAhWmTt8KHVzlAPUQ_AUoAXoECAwQAw&cshid=1601645848821&biw=1680&bih=916) might make sense, depending on size restrictions.